packages/mux-player/src/media-theme-mux.html

```diff
@@ -261,7 +261,8 @@
       line-height: 24px;
     }
 
-    .title-display {
+    :is(.title-display, media-live-button) {
+      color: inherit;
       font-size: 16px;
```

Comment:
overall great but

(blocking): `media-live-button`'s  should be 14px and vertically aligned against the red dot and the title